template.distillery/PROJECT_NAME_container.eliom

```diff
@@ -3,62 +3,58 @@
 
 (** This module defines the default template for application pages *)
 
-[%%shared
-  open Eliom_content.Html.F
-]
 
-[%%shared.start]
-
-let user_menu close user uploader = [
-  p [pcdata "Change your password:"];
-  Eba_view.password_form ~service:Eba_services.set_password_service' ();
-  hr ();
-  Eba_userbox.upload_pic_link close uploader (Eba_user.userid_of_user user);
-  hr ();
-  Eba_userbox.reset_tips_link close;
-  hr ();
-  Eba_view.disconnect_button ();
-]
-
-let%client _ = Eba_userbox.set_user_menu user_menu
+let%shared navigation_bar =
+  let nav_elts = [
+    ("Home",Eba_services.main_service);
+    ("About",%%%MODULE_NAME%%%_services.about_service);
+    ("Demo",%%%MODULE_NAME%%%_services.otdemo_service)
+  ] in
+  fun () ->
+    Eba_tools.NavigationBar.of_elt_list
+      ~elt_class:["nav";"navbar-nav"]
+      nav_elts
 
-let header ?user () =
+let%shared eba_header ?user () = Eliom_content.Html.F.(
```

Comment:
There is already the header function in Eliom_content.Html modules so it prevents potential collision.